test/e2e/network/netpol/network_policy.go

```diff
@@ -1078,6 +1078,8 @@ var _ = common.SIGDescribe("Netpol", func() {
 			ginkgo.By("Trying to connect to TCP port 81, which should be blocked by the deny-ingress policy.")
 			reachability := NewReachability(model.AllPods(), true)
 			reachability.ExpectAllIngress(NewPodString(nsX, "a"), false)
+			reachability.ExpectAllIngress(NewPodString(nsX, "b"), false)
```

Comment:
you can replace with a more generic statement, like:
```
reachability := NewReachability(model.AllPods(), true)
reachability.ExpectPeer(&Peer{}, &Peer{Namespace: nsX}, false)
```

Reply:
otherwise, the fix makes sense

Reply:
Thanks @knabben, I have added a commit with the above change.